### ISSUE TITLE
"Edit workspace" and "What is an Azure ML experiment?" is not accessible using the keyboard: A11y_ML.NET Model Builder_Environment_Keyboard.

### ISSUE THREAD

**Issue by AnKushSingh05** [dotnet/machinelearning-modelbuilder#3026]
### GitHubTags
#A11ySev1; #A11yTCS;#DesktopApp; #Win32;#Model Builder - Machine Learning for .NET Developers;#BM-MlnetModelBuilder-Win32-March25;#Win32;#WCAG2.1.1;#Keyboard;#A11yMAS; 

### Environment Details:
Application Name: VS Setup
Edition Windows 11 Enterprise
Version 24H2
OS build 26100.3476
Microsoft Visual Studio Enterprise 2022 (64-bit) - Current
Version 17.13.3

### Repro Steps:
**Prerequisites:**
Install latest Visual Studio 2022 with at least one of the .NET workloads (ASP.NET and web development, .NET desktop development, or .NET Core cross-platform development) and/or the ML.NET Model Builder component selected.

1. Launch visual studio.
2. Create a new .NET project or open an existing .NET project.
3. Press context menu button on the project in the solution explorer and Add > Machine Learning.
4. Tab till scenario and select a scenario.
5. Select training environment (Select Azure).
6. Verify that "Edit workspace" and "What is an Azure ML experiment?" is not accessible using the keyboard.

### Actual Result:
"Edit workspace" and "What is an Azure ML experiment?" is not accessible using the keyboard.

### Expected Result:
"Edit workspace" and "What is an Azure ML experiment?" should be accessible using the keyboard keys.

### User Impact:
Keyboard and screen reader users will not be able to edit the workspace.

### Attachment:

https://github.com/user-attachments/assets/78e1cac5-bd90-48bb-a608-e6895a15bd70

**Comment by AnKushSingh05** [comment]
Fixed in version: Microsoft Visual Studio Enterprise
Channel: Canary
Version: 18.3.0 Insiders [11311.1.main]

https://github.com/user-attachments/assets/ce985dfd-fd7d-4fa6-9e2b-134cfbe165be
